packages/server/src/mL/Docker.ts

```diff
@@ -76,8 +76,13 @@ export default class Docker {
         label: ["wpilib=ml"]
       }
     });
-    await Promise.all(containers.map(async (container) => this.docker.getContainer(container.Id).stop()));
-
+    await Promise.all(
+      containers.map(async (listcontainer) => {
+        const container = await this.docker.getContainer(listcontainer.Id);
+        if (listcontainer.State == "running") await container.stop();
```

Comment:
There are two declarations of .State, one in `Dockerode.ContainerInfo` and one in `Dockerode.ContainerInspectInfo` the first is a string and the second has a `Status` that is `string`, but `Running` that is `boolean`.
Should this be using the enum / dict type thing (not sure correct term for ts) so that if the backend of using the string `"running"` is changed that this will update? Or is that a problem for another time?

Reply:
I am pretty sure this is not containerInspectInfo so I dont think I can do the enum. Not sure.